source/_integrations/default_config.markdown

```diff
@@ -15,7 +15,7 @@ This integration is a meta-component and configures a default set of integration
 - [Configuration](/integrations/config/) (`config`)
 - [Counter](/integrations/counter/) (`counter`)
 - [DHCP Discovery](/integrations/dhcp/) (`dhcp`)
-- [Energy](/integrations/energy/) (`energy`)
+- [Energy](/docs/energy/) (`energy`)
```

Comment:
This is changing a valid link and should not be changed. This needs to be pointing to the integration.

Reply:
Right now the `/integrations/energy/` endpoint does not exist.
This is redirected to `/docs/energy/`
So you like to keep the `/integrations/energy/` link? Maybe to add documentation in the future?
That's fine, then I will revert the commit.
Thanks.

Reply:
It might be used in the future. At this point, we have redirected the integration page to the specific documentation, however, these links are explicitly pointing to the integration.

Reply:
Done. thx for the feedback.